src/Symfony/Component/Messenger/Command/DebugMessengerCommand.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * A console command for messenger information.
+ *
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ *
+ * @experimental in 4.1
+ */
+class DebugMessengerCommand extends Command
+{
+    protected static $defaultName = 'debug:messenger';
+
+    private $mapping;
+
+    public function __construct(array $mapping)
+    {
+        parent::__construct();
+
+        $this->mapping = $mapping;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDescription('Lists classes you can dispatch using the message bus')
```

Comment:
I suggest "message classes" or "messages" (as you say "The following messages [...]" in the output)